diagnostics/concord-ctl

```diff
@@ -10,3 +10,6 @@ cmd = b' '.join(sys.argv[1:]) + b'\n'
 s.send(cmd)
 data = s.recv(64*1024)
 print(data.decode())
+f = open("status.txt", "w")
```

Comment:
I'd suggest 2 things:
1. use `with`:
```
with open(...) as f:
    f.write(...)
```
2. It may be useful to add a parameter `-o` to set the path

Reply:
> LGTM!
> But I have a general question: why can't they do just:
> `./concord-ctl status get replica > whatever_file_they_want.txt`?
> 
> P.S.
> I've left a few minor comments.

I agree. I don't think we should add this at all. I think it is completely unnecessary. Shell redirect should be used. 

Let's try not to add options for things that are easy to do in the shell. Options should only be added for different types of diagnostics.

Reply:
1) I'll change thanks.
2) I'm not sure if we need to set a path - don't want to add more parameters to the call - I think that the root is ok.

About your suggestion I thought about it also, the only thing I can thought on is that they want it to happen every time and automatically. but I'm waiting for them to connect today to ask actually.